850.split-ambiguities/s.yaml

```diff
@@ -315,6 +330,10 @@
 - { name: swift, summpart: openstack, setname: openstack-swift }
 - { name: swift, warning: "please classify me" }
 
+- { name: swm, wwwpart: /dcat/, setname: swm-simple }
+- { name: swm, wwwpart: small, setname: swm-small }
```

Comment:
wwwpart should be more precise. http://www.informatik.hu-berlin.de/prog/swm.html is not covered.